### ISSUE TITLE
zzh8829/yolov3-tf2

### ISSUE THREAD

**Issue by aif2017** [jinfagang/yolov3_tf2#2]
hi
i think zzh8829/yolov3-tf2 implementation has bug. specialty in training. 
I test it for training just one img but loss didn't decrease properly.
what is the problem with it?

**Comment by jinfagang** [comment]
@aif2017  Yes, you were right, it has been test to be buggy.

My branch also does trainable on TensorFlow 2.0, it can only predict.

I prefer training yolov3 with pytorch repos. TensorFlow2.0 just too many bugs.
